src/guiapi/src/window.cpp

```diff
@@ -319,33 +319,35 @@ bool window_t::EventEncoder(int diff) {
         return false;
 
     if (diff > 0) {
-        capture_ptr->WindowEvent(capture_ptr, GUI_event_t::ENC_UP, (void *)diff);
+        capture_ptr->WindowEvent(capture_ptr, GUI_event_t::ENC_UP, (void *)(intptr_t)diff);
```

Comment:
Alan changed unit test to be compiled as 64bit - because of bitten apple, and this was causing error
I disagreed ... I think unit test should be as close to our platform as possible - thus 32bit

Reply:
ok